flask_admin/model/typefmt.py

```diff
@@ -44,13 +48,27 @@ def list_formatter(view, values):
     return u', '.join(text_type(v) for v in values)
 
 
+def enum_formatter(view, value):
+    """
+        Return the name of the enumerated member.
+
+        :param value:
+            Value to check
+    """
+    return value.name
+
+
 BASE_FORMATTERS = {
     type(None): empty_formatter,
     bool: bool_formatter,
     list: list_formatter,
 }
+if Enum:
+    BASE_FORMATTERS[Enum] = enum_formatter
 
 EXPORT_FORMATTERS = {
     type(None): empty_formatter,
     list: list_formatter,
 }
+if Enum:
+    BASE_FORMATTERS[Enum] = enum_formatter
```

Comment:
Needs to be `EXPORT_FORMATTERS` instead of `BASE_FORMATTERS` here. Also, maybe put them under the same IF statement?